test/fixtures/android-app/build.gradle

```diff
@@ -1,14 +1,14 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
-    ext.kotlin_version = '1.2.21'
+    ext.kotlin_version = '1.2.51'
```

Comment:
I could have presented this as a separate PR, to be honest - it allows the test fixture to be build locally on MacOS 10.15.